### ISSUE TITLE
Flip the interfaces (i.e. serve traffic over Ethernet)?

### ISSUE THREAD

**Issue by danseitz** [martinsauter/WLAN-VPN-Pi#15]
Would it be possible to configure the Pi to connect to the internet via a wireless router by configuring wpa_supplicant and then tunnel the traffic to the Ethernet port?  Basically, there is only one device I need on the VPN and it’s a bit far to the router. If I hardwire my Pi, its ability to transmit a wireless network is much inferior to my router’s

**Comment by martinsauter** [comment]
Yes, I think that could be done. Have a go at the config files that are modified by the script. Have a look what service is configured to which interface and change to fit your requirements. Sorry, don't have a ready solution for this, not my scenario.
